runtime/src/runtime_api.rs

```diff
@@ -284,7 +294,19 @@ impl_runtime_apis! {
             let mut batches = Vec::<BenchmarkBatch>::new();
             let params = (&config, &whitelist);
 
-            add_benchmark!(params, batches, system, SystemBench::<Runtime>);
+            // Note: For benchmarking Stake and Balances we need to change ExistentialDeposit to
+            // a non-zero value.
+            // For now, due to the complexity grandpa and babe aren't benchmarked automatically
+            // we should use the default manually created weights.
+            // Finally, pallet_offences have no `WeightInfo` so there's no need to benchmark it
+            // the benchmark is only for illustrative pourpuses.
```

Comment:
That since `pallet_offences` itself doesn't have a `WeightInfo` but it's the `OffenceHandler` that should report the weight, the benchmarks that are implemented inside that pallet are only to show how the benchmarking would be done.

Here's the answer I got from the Substrate team when I asked about the benchmarks for this pallet:

> `pallet_offences` itself does not have extrinsics thus does not need any weight info on its own. The benchmarks there are for informational purposes.
> The `OnOffenceHandler` returns a weight which is used for the `on_initialize` calculation, and that lives in each pallet which implements that trait.

Reply:
Ok.